[v3] Cannot find module 'ember-load-initializers' or its corresponding type declarations.

After upgrade to v3 we see following error:

`Cannot find module 'ember-load-initializers' or its corresponding type declarations.`

I think we need to add ad extra hint to find declaration

ah its maybe fixed with same PR.. because classic build brings better error :)

`Module not found: Error: Invalid "exports" target "dist/index.js" defined for "." in the package config /tmp/broccoli-1851r09KjyAsYHsR/cache-150-webpack_bundler_ember_auto_import_webpack/node_modules/ember-load-initializers/package.json, targets must start with "./"`

--- a/package.json
+++ b/package.json
@@ -9,7 +9,10 @@
     "url": "https://github.com/ember-cli/ember-load-initializers/issues"
   },
   "exports": {
-    ".": "dist/index.js"
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    }
   },
   "main": "dist/index.js",
   "repository": {

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
       "default": "./dist/index.js"
     }
   },
-  "main": "dist/index.js",
+  "main": "./dist/index.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ember-cli/ember-load-initializers.git"

